docs/src/main/sphinx/connector/non-transactional-insert.fragment

```diff
@@ -0,0 +1,13 @@
+Non-transactional INSERT
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector supports adding rows using :doc:`INSERT statements </sql/insert>`.
+By default, data insertion is performed by writing data to a temporary table.
+You can skip this step to improve performance and write directly to the target
+table. Set the catalog property ``insert.non-transactional-insert.enabled``
```

Comment:
Suggest:
Set the ``insert.non-transactional-insert.enabled`` catalog property 
or the corresponding ``non_transactional_insert`` catalog session property  to
``true``.

As is, it is missing a verb: "Set the catalog property called ``insert.non-transactional-insert.enabled`` to...